### ISSUE TITLE
Missing Npgsql Windows installer referenced in doc

### ISSUE THREAD

**Issue by EriWerBIIR** [npgsql/doc#85]
On https://www.npgsql.org/doc/installation.html#gac-installation, there is the text:

"... you can download the Npgsql Windows installer from our Github releases page"

However, there is no Npgsql Windows installer, only source releases.

1) Is there a binary Windows installer available somewhere, then the link should be updated
2) If no binary exists, but the Windows installer is still supported, perhaps this should be a separate project, and link to that?
3) If the Npgsql Windows installer is no longer supported, I think Microsoft would like to know that, since they reference is on: https://support.microsoft.com/en-us/office/import-data-from-external-data-sources-power-query-be4330b3-5356-486c-a168-b68e9e616f5a

**Comment by roji** [comment]
The Windows installer is no longer shipped since version 5.0, but previous versions (i.e. [4.1](https://github.com/npgsql/npgsql/releases/tag/v4.1.4)) is still available and should work well.

But you're right the docs aren't helpful and recent 4.1.x patch versions also don't have an MSI. I'll fix these issues.

**Comment by EriWerBIIR** [comment]
Thank you, the text you provided here would be fine.
Thanks!

**Comment by Emill** [comment]
Does this mean users cannot import data from postgresql into Excel anymore?

**Comment by roji** [comment]
FYI I've just published a new version of the MSI for [Npgsql 4.1.8](https://github.com/npgsql/npgsql/releases/tag/v4.1.8), and updated the text in [the docs](https://www.npgsql.org/doc/installation.html).

@Emill they can, just not with Npgsql 5.0. The 4.1 versions are still there, and we patch them from time to time for important things; there's no real need to have the very latest Npgsql features when using Excel. In any case, we're not going to be able to support .NET Framework forever, so it has to happen one way or another...

**Comment by Emill** [comment]
Ok so actually it's Excel then that's lagging behind, by not supporting .NET 5...

**Comment by roji** [comment]
@Emill yeah... Excel, PowerBI and various others only support ADO.NET providers installed in the GAC, which obviously no longer exists in the world of .NET Core.
